src/spurt/io/_raster.py

```diff
@@ -13,6 +13,7 @@
 import numpy as np
 import rasterio
 from numpy.typing import DTypeLike
+from typing_extensions import Self
```

Comment:
interesting, i see this is also now yelling at the [same snaphu-py code](https://results.pre-commit.ci/run/github/719773776/1742235390.oojYkatSTYWUSuL68CGxYg)